gdnative-core/src/class.rs

```diff
@@ -174,6 +175,35 @@ impl<T: NativeClass> Instance<T> {
         &self.script
     }
 
+    /// Try to downcast `T::Base` to `Instance<T>`. This safe version can only be used with
+    /// reference counted base classes.
+    pub fn try_from_base(owner: T::Base) -> Option<Self>
+    where
+        T::Base: Clone,
```

Comment:
Is this trait bound needed? It is not used by the `try_from_unsafe_base` function